src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,104 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int LENGTH = 10;
+
+    private T [] defaultArray;
+    private int size;
+
+    public ArrayList() {
+        this.defaultArray = (T[]) new Object[LENGTH];
+        this.size = 0;
+    }
+
+    private boolean isFull() {
```

Comment:
let’s move private methods to the bottom of the file, or at least after the place where we call them

Reply:
Moved to the bottom